assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -563,6 +565,176 @@ describe( 'AudienceSelectionPanel', () => {
 		} );
 	} );
 
+	describe( 'AudienceCreationNotice / AudienceCreationSuccess', () => {
+		it( 'should display an audience creation notice with both audiences', async () => {
+			const nonSiteKitAvailableAudiences = availableAudiences.filter(
+				( { audienceType } ) => audienceType !== 'SITE_KIT_AUDIENCE'
+			);
+
+			const nonSiteKitConfiguredAudiences =
+				nonSiteKitAvailableAudiences.map( ( { name } ) => name );
+
+			const nonSiteKitReportOptions = {
+				...reportOptions,
+				dimensionFilters: {
+					audienceResourceName: nonSiteKitConfiguredAudiences,
+				},
+			};
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setAvailableAudiences( nonSiteKitAvailableAudiences );
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setConfiguredAudiences( nonSiteKitConfiguredAudiences );
+
+			provideAnalytics4MockReport( registry, nonSiteKitReportOptions );
+
+			const { getByText, waitForRegistry } = render(
+				<AudienceSelectionPanel />,
+				{
+					registry,
+				}
+			);
+
+			await waitForRegistry();
+
+			expect(
+				getByText( /Create groups suggested by Site Kit/i )
+			).toBeInTheDocument();
+			document
+				.querySelectorAll(
+					'.googlesitekit-audience-selection-panel__audience-creation-notice-audience .googlesitekit-audience-selection-panel__audience-creation-notice-audience-details h3'
+				)
+				?.forEach( ( element, index ) => {
+					expect( element ).toHaveTextContent(
+						index === 0 ? 'New visitors' : 'Returning visitors'
+					);
+				} );
+		} );
+
+		it( 'should display an audience creation notice for a single audience', async () => {
+			const mixedConfiguredAudiences = [
+				'properties/12345/audiences/1',
+				'properties/12345/audiences/2',
+				'properties/12345/audiences/3', // New visitors Site Kit audience.
+			];
+
+			const mixedSiteKitReportOptions = {
+				...reportOptions,
+				dimensionFilters: {
+					audienceResourceName: availableAudiences
+						.filter( ( { name } ) =>
+							mixedConfiguredAudiences.includes( name )
+						)
+						.map( ( { name } ) => name ),
```

Comment:
Another marginal simplification which I'll apply:

```suggestion
					audienceResourceName: mixedConfiguredAudiences,
```